code/ZAS/_gas_mixture.dm

```diff
@@ -4,12 +4,6 @@ What are the archived variables for?
 	This prevents race conditions that arise based on the order of tile processing.
```

Comment:
if something like this is actually necessary to fix race conditions (especially in single-threaded conditions) then there's a deeper problem that should be fixed instead

though that's probably quite a bit outside the scope of this PR